ports/stm32/boards/stm32h743.ld

```diff
@@ -27,3 +28,12 @@ _ram_start = ORIGIN(RAM);
 _ram_end = ORIGIN(RAM) + LENGTH(RAM);
 _heap_start = _ebss; /* heap starts just after statically allocated memory */
 _heap_end = _sstack;
+
+/* Define output sections */
+SECTIONS
+{
+    .lwip_sec (NOLOAD) : {
+    . = ABSOLUTE(0x30040000);
+    *(.EthSect)
+    } >RAM_D2 AT> FLASH
```

Comment:
Does it need `AT> FLASH`?  There's nothing to load so it shouldn't be needed.

Reply:
You are right, I removed the ```AT> FLASH```